overlays/build-tools-in-build-inputs.nix

```diff
@@ -4,17 +4,112 @@ let
   inherit (import ../lib { inherit lib; }) checkMkDerivationFor;
 
   buildTools = [
+    "antlr"
+    "asciidoc"
+    "asciidoctor"
+    "autoconf"
+    "autogen"
+    "automake"
+    "automoc4"
+    "autoreconfHook"
+    "bazel"
+    "bc"
+    "bdf2psf"
+    "bison"
+    "bmake"
+    "breakpointHook"
+    "bzip2"
+    "bundler"
     "cmake"
+    "docutils"
+    "dos2unix"
+    "desktop-file-utils"
+    "docbook2x"
+    "docbook_xml_dtd_412"
+    "docbook_xml_dtd_42"
+    "docbook_xml_dtd_43"
+    "docbook_xml_dtd_44"
+    "docbook_xml_dtd_45"
+    "docbook-xsl-nons"
+    "docbook-xsl-ns"
+    "ensureNewerSourcesForZipFilesHook"
+    "extra-cmake-modules"
+    "flex"
+    "gawk"
+    "gcc"
+    "gcc10"
+    "getopt"
+    "git"
+    "gnum4"
+    "gnumake"
+    "gnused"
+    "gnustep.make"
+    "gogUnpackHook"
+    "go-md2man"
+    "gtk-doc"
+    "gzip"
+    "help2man"
+    "intltool"
+    "libtool"
+    "lit"
+    "lld"
+    "llvm"
+    "llvmPackages"
```

Comment:
Is this even a package?

Reply:
Yeah it is but maybe we want all subpackages instead. I am not sure.

Reply:
Then we would either need to list them, or adjust matching code yet again.